mackup/applications/mps-youtube.cfg

```diff
@@ -0,0 +1,6 @@
+[application]
+name = MPS Youtube
+
+[configuration_files]
+.config/mps-youtube/config
+.config/mps-youtube/playlist_v2
```

Comment:
It's not a big deal IMO, but I want to know what @lra have to say about it, because I prefer to do a separate PR adding all the obviously needed line breaks rather than just *doing it right* for this file only.